internal/system/agent/controller/http/controller.go

```diff
@@ -13,17 +13,17 @@ import (
 	bootstrapContainer "github.com/edgexfoundry/go-mod-bootstrap/v2/bootstrap/container"
 	"github.com/edgexfoundry/go-mod-bootstrap/v2/di"
 
+	"github.com/edgexfoundry/go-mod-core-contracts/v2/common"
+	"github.com/edgexfoundry/go-mod-core-contracts/v2/dtos/requests"
 	"github.com/edgexfoundry/go-mod-core-contracts/v2/errors"
-	"github.com/edgexfoundry/go-mod-core-contracts/v2/v2"
-	"github.com/edgexfoundry/go-mod-core-contracts/v2/v2/dtos/requests"
 
 	"github.com/edgexfoundry/edgex-go/internal/pkg"
-	"github.com/edgexfoundry/edgex-go/internal/pkg/v2/utils"
+	"github.com/edgexfoundry/edgex-go/internal/pkg/utils"
+	"github.com/edgexfoundry/edgex-go/internal/system/agent/application/direct"
+	"github.com/edgexfoundry/edgex-go/internal/system/agent/application/direct/config"
+	"github.com/edgexfoundry/edgex-go/internal/system/agent/application/executor"
 	"github.com/edgexfoundry/edgex-go/internal/system/agent/container"
-	"github.com/edgexfoundry/edgex-go/internal/system/agent/v2/application/direct"
-	"github.com/edgexfoundry/edgex-go/internal/system/agent/v2/application/direct/config"
-	"github.com/edgexfoundry/edgex-go/internal/system/agent/v2/application/executor"
-	v2Container "github.com/edgexfoundry/edgex-go/internal/system/agent/v2/container"
+	v2Container "github.com/edgexfoundry/edgex-go/internal/system/agent/container"
```

Comment:
I thin the alias can now be removed once merge the two `system/agent` container packages.

Reply:
Modified.